### ISSUE TITLE
[HttpFoundation] 3.3 trusted headers docs are outdated

### ISSUE THREAD

**Issue by andig** [symfony/symfony#22954]
| Q                | A
| ---------------- | -----
| Bug report?      | yes
| Feature request? |no
| BC Break report? |no
| RFC?             | yes/no
| Symfony version  | 3.3

Reading https://github.com/symfony/symfony/blob/master/UPGRADE-3.3.md I got linked to http://symfony.com/doc/current/components/http_foundation/trusting_proxies.html which opens the 3.2 version for me (local problem or website issue). Would have expected 3.3, but 3.2 is marked as current.

After switching to the 3.3 docs I'm reading:

    Request::setTrustedProxies(array('192.0.0.1', '10.0.0.0/8'));

Imho that required the header names as second parameter now.

Also, the upgrade doc says

> The Request::setTrustedHeaderName() and Request::getTrustedHeaderName() methods are deprecated, use the RFC7239 Forwarded header, or the X-Forwarded-* headers instead.

yet the examples still show

    Request::setTrustedHeaderName(Request::HEADER_FORWARDED, 'X-Forwarded');

So bottom line- 3.3 docs seem out of sync regarding this BC change?

**Comment by stof** [comment]
Documentation lives at https://github.com/symfony/symfony-docs

**Comment by xabbuh** [comment]
see symfony/symfony-docs#7952
